docs/source/_src_user_guide/state_propagation/propagation_setup/dynamics_types/mass.rst

```diff
@@ -4,7 +4,51 @@
 Mass Dynamics
```

Comment:
@DominicDirkx this page is the most "incomplete" of the entire propagation part of the user guide, mainly because the thrust guidance page is not yet available and the mass page could rely on it. Anyway, please feel free to make changes wherever you see fit!

Reply:
Indeed, there's still a bunch of open links here due to the absence of a thrust guidance page. Other than that, do you see any important issues on the page?

Reply:
No, not really, I think that for now it is good like this 🙂